src/Common/tests/System/Xml/XPath/TestsWithEncoding/MiscWithEncodings.cs

```diff
@@ -16,7 +16,7 @@ namespace XPathTests.FunctionalTests.Location.Paths
     /// </summary>
     public static partial class MiscWithEncodingTests
     {
-        public static void TestInitialize()
+        private static void TestInitialize()
```

Comment:
💭 This works, but normally I would expect to see this code in the constructor for the test class.